packages/client/src/options.ts

```diff
@@ -0,0 +1,84 @@
+import {Transport} from './transport'
+
+/**
+ * Option for the communication with InfluxDB server.
+ */
+export interface ConnectionOptions {
+  /** base URL */
+  url: string
```

Comment:
Please rename `url` to `host` according to others client:

![image](https://github.com/bonitoo-io/influxdb3-js/assets/455137/c5087f09-fd5e-4861-91c3-5ad373fd7480)

```suggestion
  host: string
```